### ISSUE TITLE
Complete Download, yet 8 of 30, duplicates found:15?

### ISSUE THREAD

**Issue by CuriousQuestioner1** [johanneszab/TumblThree#351]
Hello,
Upfront, thanks for all your work on this.

My issue is, downloading of a blog is complete (the crawl has moved onto the next queued blog), however, the Progress bar is not full and Download Videos shows "8 of 30, duplicates found: 15". I have force a complete rescan, but the other 7 videos will never download? This is on multiple blogs. Another example, download complete and shows "7 of 39, duplicates found: 13". I have trawled through many previous Issue Posts and could not find an answer.

Running version 1.0.8.68.

NSFW blog: https://karmarxx.tumblr.com/ AND https://karmarxxx.tumblr.com/ (respectively to the above).

**Comment by CuriousQuestioner1** [comment]
![capture](https://user-images.githubusercontent.com/45849474/49940608-175aa980-ff34-11e8-9dbe-fdfcb22f49a6.JPG)

The screenshot shows my settings, in addition I have Force rescan selected, but only cause I was hoping this would help capture the missing downloads when I re-queued it. It obviously did not help,

**Comment by johanneszab** [comment]
Most likely some files are inacessible (e.g. 403/404 error code), but their thumbnail/url still exists, because the image/video was removed. Then it will be counted as (possible) download, but TumblThee cannot actually download the file.

I wouldn't count too much into these values as long as you don't have any "timeout" popping up, it should be fine.

See also #25 (I think) were I actually investigated this on a smaller blog.
